resources/resincli.robot

```diff
@@ -95,6 +97,18 @@ Remove ENV variable ${variable_name} from application ${application_name}
     ${result} =  Run Process    resin env rm ${result_id.stdout} --yes     shell=yes
     Process ${result}
 
+Get public address of device ${device_uuid}
+    Run Process    resin device public-url enable ${device_uuid}    shell=yes
+    ${result} =  Run Process    resin device public-url ${device_uuid}    shell=yes
+    Process ${result}
+    Return From Keyword    ${result.stdout}
+
+Make the request to ${address}
```

Comment:
" Make the request " here is not a generic make the request, it is extremely specific to kernel test. The aim is to create a framework with helpers so that people can write their tests, so the keywords should be as clear as possible. We might want to abstract this whole kernel test into a different resource file as this isn't resin cli specific.